docs/insomnia/analytics-collected.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: article-detail
+title:  User Analytics Collected
+category: Security
+category-url: security
+---
+
+When you first download Insomnia, you were asked if you wanted to send analytics data to the Insomnia team about your Insomnia Apps performance and your behavior in the app.  For those who clicked "yes", this article is a description of which analytics are collected and sent.
```

Comment:
should we add anything here about how they can check what their status is in the settings (and how to change/remove them in the settings)?

Reply:
Done